### PR TITLE
Create escrow_utility.py

### PR DESCRIPTION
**Pull request by ObiajuluM**
I added a method for generating crypto conditions for escrows.

### CODE REVIEW

File: xrpl/utils/escrow_utility.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from cryptoconditions import PreimageSha256
```
Comment (mvadari) [inline xrpl/utils/escrow_utility.py:1]: This needs to be added to `poetry`

**Review by mvadari — CHANGES_REQUESTED** [review]
You should add tests and run the linter. Details on how to operate in this repo are in `CONTRIBUTING.md`

**Comment by ObiajuluM** [comment]
> You should add tests and run the linter. Details on how to operate in this repo are in `CONTRIBUTING.md`

I'm sorry, I don't know how to do all that. maybe you could add the method as your own commit. thanks

**Comment by khancode** [comment]
> > You should add tests and run the linter. Details on how to operate in this repo are in `CONTRIBUTING.md`
> 
> I'm sorry, I don't know how to do all that. maybe you could add the method as your own commit. thanks

@ObiajuluM We're not prioritizing this change. However, if you want this feature soon then you can do it.

**Comment by ObiajuluM** [comment]
> > > You should add tests and run the linter. Details on how to operate in this repo are in `CONTRIBUTING.md`
> > 
> > 
> > I'm sorry, I don't know how to do all that. maybe you could add the method as your own commit. thanks
> 
> @ObiajuluM We're not prioritizing this change. However, if you want this feature soon then you can do it.

Noted.

**Comment by ObiajuluM** [comment]
Hello, I created a gist for the method its lightweight and should be treated as optional, you can find it [here](https://gist.github.com/ObiajuluM/8da69c628879161c019a9256aa168d65), @mvadari i made the change from dict to TypedDict.
thanks

**Pull request closed without merging**